proposals/002-ghc-tick-tock.md

```diff
@@ -0,0 +1,131 @@
+
+With these resources, we believe that the goal of consistent six-month releases is finally attainable.
+
+The Problem
+------------
+While all users have expressed a desire to have predictable time-based releases, many commercial users, and library authors, have expressed concern that they cannot keep pace with a six-month major release cadence. In addition, GHC developers have been straining to keep pace with backports given that there have, until recently, been three active release series:
+
+ * 8.10, which is still by far the most popular supported release
+ * 9.0, which for a variety of reasons has seen less usage
+ * 9.2, which has not yet been released for long enough to be widely used
+
+Given that we are a small-but-growing community, it is important that we use
+our human resources wisely and ensure that commercial users are not burdened by
+GHC's faster release cycle. On the other hand, long release periods and long
+backport windows had a considerable cost on GHC developers, as well as making
+it harder for users to benefit from new GHC features.
+
+Proposed change
+---------------
+
+To address the above problem, we propose a "bi-modal" six-month release cadence
+with two alternating types of major releases:
+
+* *Long-term support releases*, for which the GHC team commits to provide critical backports for at least 18 months
+* *Intermediate releases*, which will continue to have critical backports only until the next release
+
+The goal of this distinction is to offer users (especially in industrial
+contexts) who need stability and predictable maintenance windows clear
+guidance.
+
+Note that the these two release types are *not* distinguished by the new
+features or changes that they include: we expect that both release types may
+contain breaking changes.   There are two reasons for this choice:
+
+* Delaying merging a feature for six months is a major drag on GHC's developers
+* If missing the intermediate release leads to a year-long delay, a consequence
+  is iterative pressure to "just delay the release a little longer".
+
+However, it is our hope that this distinction will more clearly set user
+expectations and give GHC's maintainers a clearer picture of the expected
+lifetime of release branches.  In practice, bimodal release structure reflects
+what GHC has been doing for quite some time:
+
+ * GHC 8.6 was essentially an LTS release, enjoying five minor releases over the course of nine months
+ * GHC 8.8 was comparatively shorter, having only three minor releases over seven months
+ * GHC 8.10 was quite long-lived and even had significant additions well after 8.10.1 (e.g. with the addition of AArch64/Darwin support)
+ * GHC 9.0 was something of a runt release as focus remained on stabilizing GHC 8.10's AArch64/Darwin support
+ * GHC 9.2 is still young but the GHC developers are currently planning on supporting it into 2022.
```

Comment:
```suggestion
 * GHC 9.2 is still young but the GHC developers are currently planning on supporting it into 2023.
```

?

Reply:
Oh dera, how embarrassing. I could have sworn while writing this that it was still 2021.